setup.py

```diff
@@ -66,8 +66,7 @@ def get_requirements(requirements_file='requirements.txt'):
     maintainer_email='infrastructure@lists.fedoraproject.org',
     platforms=['Fedora', 'GNU/Linux'],
     keywords='fedora',
-    packages=find_packages(exclude=('fedora_messaging.tests',
-                                    'fedora_messaging.tests.*')),
+    packages=find_packages(),
```

Comment:
Do you really want to ship the tests with the package? I think it's nice to exclude them from the production package.

Reply:
I like the idea of being able to run the tests on the system you run the code on.